contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -410,6 +434,25 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, IERC677Rece
     emit SubscriptionConsumerAdded(subscriptionId, consumer);
   }
 
+  function _cancelSubscriptionHelper(uint64 subscriptionId) private returns (uint96, uint64, address) {
+    Subscription memory subscription = s_subscriptions[subscriptionId];
+    uint96 balance = subscription.balance;
+    uint64 completedRequests = 0;
```

Comment:
Counting requests is different to Automation and not what we discussed so far. I think it will be more consistent to just count total spending per sub.

Reply:
By utilizing a storage variable that we already have we are avoiding adding one extra storage write per fulfillment.

Curious to hear more words on how using total spending would be more consistent. I could see the argument for that total spending gives us more granularity, but is that something we would actually use?

Reply:
Is number of requests gameable by doing extremely cheap requests and not really mitigating the reason this was implemented in the first place?

Reply:
We will configure `subscriptionDepositCompletedRequests` based on the cheapest possible request, so that it will still serve the purpose of deterring attacks